pages/playlist/podcast/[id]/index.vue

```diff
@@ -10,6 +11,19 @@ const { data: tracks, pending: tracksPending } = usePodcastTracks({
   id: collectionId,
 });
 
+const onPressPlay = () => {
+  if (tracks.value) {
+    setQueue(tracks.value);
+  }
+};
+
+const onPressShuffle = () => {
+  const shuffledTracks = usePodcastShuffle(collectionId).data;
```

Comment:
Could you also add some error handling? Just wrap it into a try-catch and log any error to the console and leave a to-do here.

Reply:
Done and pushed